Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -261,9 +272,12 @@ private fun processCLib(flavorName: String, cinteropArguments: CInteropArguments
 
     val imports = parseImports(allLibraryDependencies)
 
-    val library = buildNativeLibrary(tool, def, cinteropArguments, imports)
+    val j2objcIndexerResult = IndexerResult(J2ObjCNativeIndex(jarClassNodes), CompilationWithPCH(emptyList<String>(), Language.J2ObjC))
+    val index:IndexerResult = if (language == Language.J2ObjC) j2objcIndexerResult
```

Comment:
Doesn't doing it this way mean that the Objective-C data is going to be computed no matter what version of cinterop is running?

Reply:
Yes, I will have to change that. Thanks for this catch.

Reply:
Done.